lib/routes/account.js

```diff
@@ -1007,7 +1007,7 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
       config: {
         auth: {
           mode: 'optional',
-          strategy: 'sessionToken'
+          strategy: 'sessionTokenWithDevice'
```

Comment:
Perhaps the biggest unanswered question this PR leaves is which endpoints should fail if they receive an expired session token.

If the answer is "all of them" (which I'm in favour of btw), it means we can simplify a bunch of stuff both here and in the db repo by eliminating some strategies/methods/endpoints/stored procedures. If not, we may still be able to eliminate one of them depending on the remaining usage.

In terms of endpoints that are changed here, as it stands I've done it for everything except `/session/destroy` on the basis that even an expired session should be able to delete itself. But I also can't see how it would get to that point if all the other endpoints fail, hence my suspicion that every endpoint should fail and that we can clean up the authentication strategies and related db code.